### DISCUSSION TITLE
extending theme

### DISCUSSION THREAD

**Discussion by tasiotas** [unocss/unocss#788] [Q&A] [answered]
Hi,

I just started using uno in Nuxt.
I defined my theme like this:

``` js
  unocss: {
    theme: {
      colors: {
        'primary': '#0e0e0e',
        'primary-light': '#1f1f1f',
        'accent': '#ea0ad7',
        'background-light': '#F5F5F7',
      },
      variables: {
        'navbar-height': '100px',
      },
    },
  },
```
But then trying to use 'navbar-height', style isnt generated.
``` html
<div class="mt-navbar"></div>
```
I can get color working, but nothing else.
Thanks for help

**Comment by zyyv** [comment, marked as answer]
If you want use variables, try to `mt-$navbar`, it will be genrate a css variable.

Or, maybe you can use shortcuts instead of it.
```
shortcuts: [
  { 'mt-navbar': 'mt-100px' }
]
```

**Reply by tasiotas** [reply]
Thanks for help.

So I figured out I can define variables in extra .css file, then reference them like you mentioned with `mt-$navbar`.

All good, thanks!

**Reply by Alberto0738gh** [reply]
Hey
